e2e/tests-dfx/certified_info.bash

```diff
@@ -18,7 +18,7 @@ teardown() {
     dfx_start
     dfx canister create hello
     assert_command dfx canister info "$(dfx canister id hello)"
-    assert_match "Controllers: $(dfx identity get-wallet) Module hash: None"
+    assert_match "Controllers: $(dfx identity get-principal) Module hash: None"
```

Comment:
Shouldn't both the wallet and the identity's principal be controllers?

Reply:
No, because --wallet was not specified.

Reply:
Ok, let's discuss then, because I think there's a disconnect here.  After creating a canister without passing any particular parameters (as is done here), I would expect both the wallet and the dfx identity to be controllers.

Reply:
@krpeacock thoughts?